ha/k8s_setup/ha_start.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import argparse
+from subprocess import Popen, PIPE # nosec
+import sys
+
+
+my_parser = argparse.ArgumentParser(prog="ha_start",
+                                    description='Entrypoint for HA containerization',
+                                    epilog='Fault tolerance is containerized! :)'
+                                    )
+
+my_parser.add_argument('-s', help='service_name', required=True, action='store')
+my_parser.add_argument('-c', help='config file', required=False, action='store')
```

Comment:
config file is mandatory right ? required=True ?

Reply:
No, config file is not mandatory in our case.